README.md

```diff
@@ -76,7 +76,7 @@ autocannon({
 Start autocannon against the given target, options:
 
 * `url`: the given target, mandatory
-* `connections`: the number of concurrent connections
+* `connections`: the number of concurrent connections, mandatory
```

Comment:
You use 10 in the readme examples, I think using that is a good idea as it gives a clue to users about what to expect?